public/js/user.js

```diff
@@ -1,8 +1,8 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 function updateUser(id) {
   const floorEl = document.getElementById('floor-input')
-  const floor = parseInt(floorEl.value)
-  if (!isNaN(floor) && (floor < 3 || floor > 18)) {
+  const floor = floorEl.value === '' ? null : parseInt(floorEl.value)
+  if (floor != null && (isNaN(floor) || floor < 3 || floor > 18)) {
```

Comment:
What do you think a bit less complex version?
```suggestion
  const floor = parseInt(floorEl.value)
  if (floorEl.value !== '' && (floor < 3 || floor > 18)) {
```

Reply:
This seems correct, and is indeed simpler. Will test locally and change it if it works fine.

Reply:
My small issue with this is that when the user enters some random string, it passes all the validation and gets saved as `null`. It's just a minor UX bug tho because it actually saves `null` in the database.

Reply:
Yes, I was thinking about the alpha characters and their behaviour, but then I checked the input type is set to number and only [a minority](https://caniuse.com/#feat=input-number) of the browsers does not support it.

Reply:
I was referring to the fact that Firefox and Safari allows alpha characters in number input fields, and we should handle them somehow. We could type-check the input and respond accordingly, but I think the current implementation is fine as-is.